packages/exo/src/exo-makers.js

```diff
@@ -62,11 +62,28 @@ export const initEmpty = () => emptyRecord;
  * Each property is distinct, is checked and changed separately.
  */
 
+/**
+ * @callback Revoker
+ * @param {object} exo
+ * @returns {boolean}
+ */
+
+/**
+ * @typedef {Record<FacetName, Revoker>} RevokerKit
+ */
+
+/**
+ * @callback GetRevoker
+ * @param {Revoker | RevokerKit} revoke
```

Comment:
Maybe consider always returning a `RevokerKit`, potentially with just a single entry for `self`, so that it parallels the `this.facets` record?

I think that would be worthwhile to avoid needing to complicate this parameter's type.

Reply:
I'm gonna try a different strategy: Just have a one argument `revoke` function for both cases. For the kit case, the revoke function has to do a trivial search over all the facets. 
   * Downside: This does decrease the separation of powers: Rather than needing both the instance and a facet-specific revoke function to revoke, all you'd need is the instance and a class-kit-specific revoke function that covers all the facets.
   * Upside: The API is simpler both statically and dynamically.
   * Upside: While working on https://github.com/Agoric/agoric-sdk/pull/8008 , I discovered that virtual storage is organized differently, with a single cache map across all facets, with facet cross-talk suppressed by other means. This would make my original API harder to implement than the one I'm suggesting here.

Please let me know what you think.

Reply:
Even more radical simplification at https://github.com/endojs/endo/pull/1668

Reply:
After talking to both @michaelfig and @FUDCo , the design we settled on is a `receiveRevoker` function (just a renaming of the confusingly named `getRevoker` function discussed above) that receives a `revoke` function with a single exo argument. When called with an exo instance made by this class, that instance is revoked. When called with an exo facet make by a class kit, only that facet is revoked.

I know of no remaining controversy about this design, so I'm resolving this conversation as Done.